editor.js

```diff
@@ -3,7 +3,7 @@
 	Extension Editor for XKit 7
 	Version 1.0
 
-	(c) 2011 - 2013 STUDIOXENIX
+	(c) 2011 - 2015 STUDIOXENIX & New-XKit Team
```

Comment:
Probably won't need to worry about relicensing, but a CLA would still be something to look into.

How about:

> (c) 2011 - 2014 STUDIOXENIX
> (c) 2015 New XKit Team and Contributors (https://github.com/new-xkit/XKit/contributors)